infinite_tracing/newrelic-infinite_tracing.gemspec

```diff
@@ -72,7 +72,7 @@ Gem::Specification.new do |s|
   s.summary = 'New Relic Infinite Tracing for the Ruby agent'
 
   s.add_dependency 'newrelic_rpm', NewRelic::VERSION::STRING
-  s.add_dependency 'grpc', '~> 1.34'
+  s.add_dependency 'grpc', '~> 1.59'
```

Comment:
Would this effectively make new versions of the 8T gem require Ruby 2.5 and above, since the required `grpc` version is one that doesn't work with Ruby 2.5 in our testing?

Reply:
This change does not alter anything functionally from a Bundler / Ruby perspective. It still tells Bundler to grab any 1.x version, just the same as before. The only real impact this has is that the `~> 1.59` string will appear in the gemspec file and out on the RubyGems.org listing and convey to a human reader that we have tested with 1.59 and signed off on it.

Reply:
Scratch that, @kaylareopelle. I tested it and you are right. The new constraint is effectively saying `>= 1.59 and < 2.0`, which is imposing a newer version requirement. We don't want that. I will revert this.

Reply:
reversed with [077f66a](https://github.com/newrelic/newrelic-ruby-agent/pull/2306/commits/077f66ae2d2162de621a30dce90928e95fc86b72)